### PR TITLE
feat(markdown-and-latex): add `markview.nvim`

### PR DESCRIPTION
**Pull request by mehalter**
## 📑 Description

Adds [`markview.nvim`](https://github.com/OXY2DEV/markview.nvim)

### CODE REVIEW

File: lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua [hunk 1]
```diff
@@ -0,0 +1,9 @@
+return {
+  "OXY2DEV/markview.nvim",
+  ft = { "markdown", "markdown.mdx" },
+  dependencies = { "nvim-treesitter/nvim-treesitter" },
```
Comment (ALameLlama) [inline lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua:4]: on their docs, they also have `nvim-tree/nvim-web-devicons` as a dep, should we include this here too?

**Review by ALameLlama — CHANGES_REQUESTED** [review]

File: lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua (hunk 1 above)
Comment (mehalter) [reply lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua:4]: It's fine without it because it will load on module

Comment (ALameLlama) [reply lua/astrocommunity/markdown-and-latex/markview-nvim/init.lua:4]: sweet

**Review by ALameLlama — APPROVED** [review]
LGTM

**Pull request merged**